src/main/groovy/com/profitbricks/sdk/Commands.groovy

```diff
@@ -31,10 +31,21 @@ class Commands {
      * gets the attached volumes for a given server
      *
      * @param server the server with the attached volumes
-     * @return a list of volume ids
+     * @return a list of attached volumes
      */
-    static final List<String> attachedVolumes(final Server server) {
-        API.get(requestFor("${server.resource}/${server.id}/volumes"))?.data?.items?.collect{it.id}
+    static final List<Volume> attachedVolumes(final Server server) {
+        API.get(requestFor("${server.resource}/${server.id}/volumes"))?.data?.items?.collect{new Volume().from(it) as Volume}
+    }
+    
+    /**
+     * This will retrieve the properties of an attached volume.
+     *
+     * @param server the server with the attached volume
+     * @param volume an existing volume
+     * @returns the volume attached
+     */
+    static final Volume attachedVolume(final Server server,final Volume volume) {
```

Comment:
Replaced volume with a string volumeId so this can be more useful, if you still think its unnecessary we can remove it.